### PR TITLE
Support Python 3

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
@optilude @wichert Could you add me ("icemac") as owner on PyPI for this package?

File: CHANGES.rst [hunk 1]
```diff
@@ -2,10 +2,11 @@ Changelog
 =========
 
 
-2.0b1 - Unreleased
-------------------
+3.0 - Unreleased
+----------------
```
Comment (mgedmin) [inline CHANGES.rst]: Version 2.0 final was never released.  Are we skipping it entirely?  Why?

Comment (icemac) [reply CHANGES.rst]: You are right, 2.0 should be enough.

**Comment by wichert** [comment]
Why is this removing the license file?

**Comment by wichert** [comment]
@icemac I've added you on pypi

**Comment by icemac** [comment]
@wichert Thank you.

**Pull request merged**